packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -752,6 +752,10 @@ const Component = CoreView.extend(
           id: 'ember-views.event-dispatcher.mouseenter-leave-move',
           until: '4.0.0',
           url: 'https://emberjs.com/deprecations/v3.x#toc_component-mouseenter-leave-move',
+          for: 'ember-source',
+          since: {
+            enabled: '3.21.0',
```

Comment:
I think the "real" enabled versions are in here:

https://github.com/emberjs/ember.js/blob/master/packages/%40ember/deprecated-features/index.ts#L6-L19

Can you update each of these to match those versions?